demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1152,6 +1234,16 @@ export class DemoMeetingApp
       return (document.getElementById(id) as HTMLInputElement).checked;
     }
 
+    function getSelectedValues(id: string): string {
```

Comment:
Short comment explaining intention of this function would be helpful

Reply:
Updated in latest revision